### DISCUSSION TITLE
Ideal maximum transmit power in Sionna RT

### DISCUSSION THREAD

**Discussion by charaf412** [NVlabs/sionna#356] [General]
Hello,

What is the ideal maximum transmit power used in Sionna RT for this [tutorial](https://nvlabs.github.io/sionna/examples/Sionna_Ray_Tracing_Introduction.html)

Is a maximum transmit power of Pmax = 40 dBm a suitable value?

Thank you for your answer in advance!

**Comment by jhoydis** [comment]
Hi @charaf412,

The transmit power is assumed to be equal to one and we scale the noise power to obtain a desired signal-to-noise ratio. Note that the channel frequency responses also need to be normalized for this (see cell [40]):
```py
self._channel = OFDMChannel(
                            self._channel_model,
                            self._pusch_transmitter.resource_grid,
                            normalize_channel=True,
                            return_channel=True)
```

Setting an explicit transmit power does not make sense for link-level simulations. This is something that you would rather do for system-level simulations that account for power control, link adaptation, etc.

Hope that answers your question.

**Reply by charaf412** [reply]
Hello,
thank you so much for your answer.
In the scenario I am working on, I also have mobile users.
Is the transmit power assumed to be equal to 1 dBm for both base station and mobile users, or only for base station?
If it's only for base station, what is the transmit power for mobile users?

**Reply by jhoydis** [reply]
By default, all transmit signals are assumed to have unit variance. There is no notion of Watt or dBm unless you take the symbol rate into account. You can simply verify this by computing the variance of the tensor of modulated symbols.
